corehq/apps/app_manager/models.py

```diff
@@ -4082,7 +4099,13 @@ def wrap(cls, data):
                 data['text_input'] = 'native' if data['native_input'] else 'roman'
             del data['native_input']
 
-        should_save = False
+        if 'build_langs' in data:
+            if (data['build_langs'] != data['langs']) and ('build_profiles' not in data):
+                data['build_profiles'] = {uuid.uuid4().hex : BuildProfile(name=', '.join(data['build_langs']), langs=data['build_langs'])}
```

Comment:
this line is pretty long (I guess not long enough to tick off @dimagimon). consider wrapping or restructuring the code